Configuration/Geometry/cmsExtendedGeometry2026D98.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
```

Comment:
why is this file needed? Seems a copy of `Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2026D98.xml`